helm/libero-reviewer/templates/deployment-continuum-auth.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: {{ include "libero-reviewer.fullname" . }}-continuum-auth
+  labels:
+    app.kubernetes.io/name: {{ include "libero-reviewer.name" . }}
+    helm.sh/chart: {{ include "libero-reviewer.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: client
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "libero-reviewer.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+      app.kubernetes.io/component: client
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "libero-reviewer.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+        app.kubernetes.io/component: client
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}-continuum-auth
+          image: "{{ .Values.images.continuumAuth.repository }}:{{ .Values.images.tag }}"
+          imagePullPolicy: {{ .Values.images.pullPolicy }}
+          env:
+            - name: RABBITMQ_URL
+              value: "{{ .Release.Name }}-rabbitmq"
+            - name: CONTINUUM_LOGIN_URL
+              value: "{{ .Values.continuumLogin.url }}"
+            - name: CONTINUUM_LOGIN_PORT
+              value: "{{ .Values.continuumLogin.port }}"
+            - name: CONTINUUM_LOGIN_JWT_SECRET
+              value: "{{ .Values.continuumLogin.secret }}"
```

Comment:
this looks like it has to be a secret that is created out-of-band (until we have Vault synchronization in the future):
https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables

Reply:
Shall we set k8s secrets as separate pr...

Reply:
You can start to use them in another PR; but you can't set them in a PR because they are secret; they need to be set manually.